### DISCUSSION TITLE
Email test ?mailripv2

### DISCUSSION THREAD

**Discussion by Sharkleur** [DrPython3/MailRipV2#9] [General]
hi mate, i don't understand why i don't receive emails test of smtp server when I have hits.
can u explain me plz ?
thx u your smtp cracker is very nice!

**Comment by DrPython3** [comment]
Hey there!

There are many reasons why you may not receive any test e-mail. Most common reasons are:

- You are using proxys and the proxys are blacklisted or flagged for spam.
- The SMTP does not accept e-mail sending using third party applications (e.g. GMAIL has a certain security feature for this).
- The e-mail templates are raising flags.
- The SMTP account has been flagged for spam or has reached its sending limit.
- The receiving address rejects the e-mail sent by Mail.Rip.

I have tested the e-mail sending feature several times and it is working though the delivery rate is sadly low at the moment. Regarding this, I am already working on an update, but I cannot promise a release date of that update yet. You may try editing the e-mail templates and not using proxys - that should provide a higher success rate for a short time.

Nevertheless, watch out for further updates.

Best regards,
DrPython3

**Reply by Sharkleur** [reply]
Thank's you for your answer, 
but there is something that i don't understand : what i need to do with "email templates", i don't understand their functions.
thx !

**Reply by DrPython3** [reply]
When downloading Mail.Rip V2 the code comes with five text files called "email_template(...).txt". After a valid SMTP login was found, Mail.Rip randomly chooses one of these files, reads the text it contains and uses that content for the test message which will be send to you. So, those text files contain the drafts of available test e-mails.

The original files contain all the same text, but you can modify them as often as you want. Before you start using the checker / cracker, you should edit every email_template-file. Each file should contain a unique text then - that way Mail.Rip can send out five different test e-mails. This will already increase the delivery rate. For best delivery results you may want to use a spam check tool (there are several free ones available on the web) to make sure the content of your modified templates won't be detected as spam.

Thereby regard the original text files contain five "placeholders" starting with $. Those will be replaced with the certain data of the found SMTP. For example: ${smtp_email} will be replaced with the e-mail address of the found login. You just have to insert the placeholder "${smtp_email}" without the " to your modified email_templates. I have attached an example of a modified template file for you - just compare it with one of the original files and all should be clear.

Good luck & have a nice weekend!

[email_template1.txt](https://github.com/DrPython3/MailRipV2/files/6176405/email_template1.txt)

**Comment by DrPython3** [comment]
I have uploaded the updated version (release 9) now. E-mail test should work better now. Nevertheless, edit email templates before using Mail.Rip. Therefor, this topic is closed. Best, DrPython3
